app/services/menu/loader.rb

```diff
@@ -34,7 +34,7 @@ def self.load
             menu.item :roles,            :caption => N_('Roles')
           end
           menu.divider
-          menu.item :bookmarks,          :caption => N_('Bookmarks')
+          menu.item :bookmarks,          :caption => N_('Bookmarks') if User.current && User.current.can?(:edit_bookmarks)
```

Comment:
question is - what if the user is an admin? shouldn't they see everything? also, "my bookmarks" are very badly defined, there's no way to define it using taxonomies, etc...
I think we should solve the CVE, and think about redesigning bookmarks' permissions